core/src/style/spriteStyle.h

```diff
@@ -49,8 +47,5 @@ class SpriteStyle : public Style {
     SpriteStyle(std::string _name, GLenum _drawMode = GL_TRIANGLES);
 
     virtual ~SpriteStyle();
-    
-    // FIXME : only one sprite tile can be processed at a time
-    static MapTile* s_processedTile;
```

Comment:
Yes, me too!

On Mon, Jul 13, 2015 at 6:30 PM, Matt Blair <notifications@github.com>
wrote:

>> @@ -49,8 +47,5 @@ class SpriteStyle : public Style {
>>      SpriteStyle(std::string _name, GLenum _drawMode = GL_TRIANGLES);
>>  
>>      virtual ~SpriteStyle();
>> -    
>> -    // FIXME : only one sprite tile can be processed at a time
>> -    static MapTile* s_processedTile;
> Glad this can go away, this has made me anxious for a while XD
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tangrams/tangram-es/pull/172/files#r34519578